README.md

```diff
@@ -58,6 +56,14 @@ On Ubuntu and Raspberry Pi OS you can get the libraries by running
 $ sudo apt-get install libssl-dev libcurl4-openssl-dev liblog4cplus-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev gstreamer1.0-plugins-base-apps gstreamer1.0-plugins-bad gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly gstreamer1.0-tools
 ```
 
+#### Building the Sample Code
+
+The GStreamer and JNI examples are NOT built by default.  If you wish to build JNI you MUST add -DBUILD_JNI=TRUE and for GStreamer examples you MUST add -DBUILD_JNI=TRUE.  To build both:
```

Comment:
Can you modify the line to this: 

The GStreamer and JNI examples are NOT built by default.  If you wish to build JNI you MUST add `-DBUILD_JNI=TRUE` and for GStreamer plugin and samples, you MUST add `-DBUILD_GSTREAMER_PLUGIN=ON`

Reply:
Change made as new commit.  you should be able to do a merge squash when you accept it so I didn't squash it

Reply:
Not sure why, but I do not see the changes you made. The lines still are the same?